front/app/components/FilterBoxes/TopicsFilter.tsx

```diff
@@ -32,21 +50,26 @@ const Topic = styled.button`
   margin-bottom: 8px;
   cursor: pointer;
   user-select: none;
-  border: solid 1px ${colors.divider};
+  border: solid 1px transparent;
   border-radius: ${(props) => props.theme.borderRadius};
   transition: all 80ms ease-out;
 
+  ${isRtl`
+        text-align: right;
+        direction: rtl;
+    `}
+
   &:not(.selected) {
     &:hover {
-      color: ${({ theme }) => theme.colors.tenantSecondary};
-      border-color: ${({ theme }) => theme.colors.tenantSecondary};
+      color: ${({ theme }) => theme.colors.tenantPrimary};
+      border-color: ${({ theme }) => theme.colors.tenantPrimary};
     }
   }
 
   &.selected {
     color: #fff;
-    background: ${({ theme }) => theme.colors.tenantSecondary};
-    border-color: ${({ theme }) => theme.colors.tenantSecondary};
+    background: ${({ theme }) => theme.colors.tenantPrimary};
+    border-color: ${({ theme }) => theme.colors.tenantPrimary};
 
     &:hover {
       background: ${({ theme }) => darken(0.15, theme.colors.tenantSecondary)};
```

Comment:
I think we should rather use the same styles as we do for the status and sorting buttons. So light gray background, no border on hover. Wdyt?

Reply:
Yeah good call! Updated them :+1: